kratos/utilities/mortar_utilities.h

```diff
@@ -505,7 +505,7 @@ class MortarUtilities
      */
 
     template< unsigned int TNumNodes, unsigned int TDim>
-    static inline BoundedMatrix<double, TNumNodes, TDim> ComputeTangentMatrix(const GeometryType& ThisNodes) {
+    static inline BoundedMatrix<double, TNumNodes, TDim> ComputeTangentMatrix(GeometryType& ThisNodes) {
```

Comment:
@loumalouomega minor comment, but why do you need to remove the const here? I guess that some of the methods you are calling do not have the const in their declaration, but I suspect they might be missing it (?)

Reply:
The problem is what you say, and without removing it, it does not compile

Reply:
yes, but which method requires it? I'd like to understand what are you trying to do that is not const, because it looks like a const is missing somewhere to me (in the core, not in this PR).

Reply:
The problem is that in some problems the geometry can not be defined as const (the input is a geometry)